### DISCUSSION TITLE
Love it when GitHub support completely ghosts me🥰

### DISCUSSION THREAD

**Discussion by SidmoGoesBrrr** [community/community#65280] [GitHub Education]
### Hi everyone,

It has been 5 weeks since I've been making posts of this. I uploaded valid ID with date name college name. I uploaded fee receipt as well as the student portal and still not response+auto reject. I made multiple tickets and surprise surprise: no response again 
![Screenshot_2023-08-29-13-00-37-033_com android chrome](https://github.com/community/community/assets/60550481/64d07ced-116a-41d7-886b-9fa34a3cb78a)
![Screenshot_2023-08-29-13-00-40-340_com android chrome](https://github.com/community/community/assets/60550481/d33d6615-b806-4b5e-bc25-6ede5e154864)
![Screenshot_2023-08-29-13-00-46-061_com android chrome](https://github.com/community/community/assets/60550481/880bd20e-3928-404f-8550-91422d5c2648)
![Screenshot_2023-08-29-13-00-48-191_com android chrome](https://github.com/community/community/assets/60550481/8bfe0819-a593-48cf-9b1b-49c477f78441)

### A suggestion...(optional)

Please respond to support tickets.

**Comment by samus-aran** [comment]
Hi @SidmoGoesBrrr  👋

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐ We are seeing a higher volume of student developer pack inquiries coming through. We are working our way through addressing these as fast as possible. 

From the sounds of it you're stating that your school doesn't supply academic emails. That sounds like the school info we have on record needs updating. Since you have a ticket open I'll update this and forward it onto the internal team to review the school's details. 

If I have any updates I'll update here 👍

**Reply by SidmoGoesBrrr** [reply]
Okay that would be great

**Reply by SidmoGoesBrrr** [reply]
Nothing has been done to this as of now. 
![image](https://github.com/community/community/assets/60550481/11559a6d-2746-4e28-a887-37db8448f596)
Infact, I reapplied and got rejected again with the same reason :(
Please sort this out ASAP I rely on github benefits tools.

**Reply by Neon-Cube** [reply]
I am experiencing a similar problem. Every time I apply, It gets rejected for requirements that either are not clear or have already been satisfied.
The latest one which is the Name and Short Bio one is unclear on the second part. What is the short bio supposed to contain? What information do I have to give? It wasn't after rummaging through the forums that I found out you also have to set your location to your school's location.
Just like the Author, I rely on GitHub Student Developer Pack Tools, so I'd appreciate it if you people sorted things out.

**Comment by SidmoGoesBrrr** [comment]
Any updates @samus-aran ?

**Comment by samus-aran** [comment]
[Closing in favour of the latest discussion](https://github.com/orgs/community/discussions/68909) 👍
